### ISSUE TITLE
Organizing pneumonia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6701]
**Preferred term label:**
Organizing pneumonia

**Synonyms**

**Definition (free text, please give PubMed ID)**
Organizing pneumonia manifests as a histologic pattern characterized by loose plugs of connective tissue in the airspaces and distal airways. Interstitial inflammation and fibrosis are minimal or absent. Cryptogenic organizing pneumonia, or COP, is a distinctive clinical disorder among the idiopathic interstitial pneumonias (5), but the histologic pattern of organizing pneumonia is encountered in many different situations, including pulmonary infection, hypersensitivity pneumonitis, and collagen vascular dis-
eases. Airspace consolidation is the cardinal feature of organizing pneumonia on chest radiographs and CT scans. In COP, the
distribution is typically subpleural and basal and sometimes bronchocentric. Other manifestations of organizing pneumonia include groundglass opacity, tree-in-bud pattern, and nodular opacities.

PMID:18195376

Comment
See Figure 45 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
